src/ConfirmProvider/ConfirmProvider.test.js

```diff
@@ -33,10 +35,10 @@ function ConfirmDialog() {
 }
 
 describe("ConfirmProvider", () => {
-  // tests in dark mode
-  it("onClick button should show dialog with title, description,cancellation text and Confirmation text with appropriate colors and bgcolor in dark mode", () => {
+  // tests in light mode
+  it("onClick button should show dialog with title, description,cancellation text and Confirmation text with appropriate colors and bgcolor in light mode", () => {
```

Comment:
These tests are doing too much, highlighted by the fact that the test name is three lines long. 

A test should really confirm one behaviour.

For instance, I would expect the following tests:
* hasDefaultText
* supportsLightMode
* supportsDarkMode

We don't really need to test the behaviour of material-ui-confirm, just the defaults we've set, so you can do away with the cancel stuff.